packages/model-viewer/package.json

```diff
@@ -36,6 +36,7 @@
     "prepare": "if [ ! -L './shared-assets' ]; then ln -s ./node_modules/@google/model-viewer-shared-assets ./shared-assets; fi",
     "build": "tsc && rollup -c --environment NODE_ENV:production",
     "build:dev": "tsc && rollup -c --environment NODE_ENV:development",
+    "build:release": "mkdir -p release; cp LICENSE POLYFILLS.md README.md package-lock.json package.json screenshot.png release; cp -r dist release/dist; cp -r lib release/lib",
```

Comment:
Let's discuss changing our release process in a separate PR

Reply:
apologies - this was a later push to the branch, wasn't intended to be included